src/js/request_scheduler.ts

```diff
@@ -229,6 +229,7 @@ class RequestScheduler {
             if (!signin.checkTooManyRedirects(url, req) ) {
                 console.log( 'Unknown error fetching ' + url );
             }
+            failure_callback(url);
```

Comment:
how embarassing!
this should not be in the category change, but I will definitely merge it if you send it as another one.